packages/eas-cli/src/user/User.ts

```diff
@@ -1,16 +1,17 @@
 import gql from 'graphql-tag';
 
-import { apiClient, graphqlClient } from '../api';
-import { Account } from './Account';
+import { apiClient } from '../api';
+import { graphqlClient } from '../graphql/client';
+import { UserFragments, UserQuery } from '../graphql/queries/UserQuery';
+import { User } from '../graphql/types/User';
+import { UserAccount } from './Account';
 import { getSession, setSessionAsync } from './sessionStorage';
 
-export interface User {
-  userId: string;
-  username: string;
-  accounts: Account[];
-}
+export type CurrentUser = Pick<User, 'id' | 'username'> & {
```

Comment:
```suggestion
export type User = Pick<User, 'id' | 'username'> & {
```

Reply:
Why revert?
cc @dsokal

Reply:
@wkozyra95 We decided to rename it to avoid confusion with the `User` type from graphql directory.

Reply:
Unless you have a really strong argument why we shouldn't rename this, it'd leave it as Bartek proposed.

Reply:
We will type from `user/User.ts` almost exclusively in the entire codebase and CurrentUser is not a very good name, 
In my opinion, a type should describe structure not identify exact value, in this case, a variable of that type can only have one specific value. 

Also, I don't see a problem with the confusion of those two types, in another case I would probably agree, but graphql type names will collide quite often with types we define in cli and it most cases they won't be used directly but like here via some other type with a more specific scope.

One more thing I would prefer here is to define User the way it was not with `Pick`, ts will check if types are correct either way.

Reply:
discussed offline with @dsokal , let's revert back to User.